### PR TITLE
JOSS-Paper

### PR DESCRIPTION
**Pull request by Yurlungur**
## PR Summary

This is the long-awaited draft of the paper submission for JOSS. Please make any changes you think appropriate to this branch. I will merge into main and then submit to JOSS in a few days, at which point you may make changes via a new pull request.

**Important:** If you believe someone (including yourself!) is missing from the author list, now is the time to add them.

## PR Checklist

- [ ] Adds a test for any bugs fixed. Adds tests for new features.
- [ ] Format your changes by using the `make format` command after configuring with `cmake`.
- [ ] Document any new features, update documentation for changes made.
- [ ] Make sure the copyright notice on any files you modified is up to date.
- [ ] After creating a pull request, note it in the CHANGELOG.md file.
- [ ] LANL employees: make sure tests pass both on the github CI and on the Darwin CI

If preparing for a new release, in addition please check the following:
- [ ] Update the version in cmake.
- [ ] Move the changes in the CHANGELOG.md file under a new header for the new release, and reset the categories.

### CODE REVIEW

File: joss-paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+    affiliation: "1, 2"
+    corresponding: true
+  - name: Daniel A. Holladay
+    affiliation: "2, 3"
+  - name: Jeffrey H. Peterson
+    affiliation: 4
+  - name: Christopher M. Mauney
+    affiliation: 5
+  - name: Richard Berger
+    affiliation: 3
```
Comment (rbberger) [inline joss-paper/paper.md:22]:
```suggestion
    orcid: 0000-0002-3044-8266
    affiliation: 3
```

**Comment by Yurlungur** [comment]
You can look at the compiled PDF clicking on the `Actions` tab at the top, then navigating to `Generate JOSS paper` on the left. Finally, click on the most recent workflow. A pdf is available for download in the artifacts section below the workflow summary.
image

File: joss-paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    affiliation: 9
+  - name: Mariam Gogilashvili
+    affiliation: "1, 2, 7, 10"
+  - name: Joshua C. Dolence
+    affiliation: "1, 2"
+  - name: Chad D. Meyer
+    affiliation: 11
+  - name: Sriram Swaminarayan
+    affiliation: 3
+  - name: Christoph Junghans
```
Comment (junghans) [inline joss-paper/paper.md:42]:
```suggestion
  - name: Christoph Junghans
    orcid: 0000-0003-0925-1458
```

File: joss-paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+  - performance portability
+authors:
+  - name: Jonah M. Miller
+    orcid: 0000-0001-6432-7860
+    affiliation: "1, 2"
+    corresponding: true
+  - name: Daniel A. Holladay
+    affiliation: "2, 3"
+  - name: Jeffrey H. Peterson
+    affiliation: 4
```
Comment (jhp-lanl) [inline joss-paper/paper.md:18]:
```suggestion
  - name: Jeffrey H. Peterson
    orcid: 0000-0001-9425-4674
    affiliation: 4
```

File: joss-paper/paper.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+  - Python
+  - equations of state
+  - thermodynamics
+  - performance portability
+authors:
+  - name: Jonah M. Miller
+    orcid: 0000-0001-6432-7860
+    affiliation: "1, 2"
+    corresponding: true
+  - name: Daniel A. Holladay
```
Comment (dholladay00) [inline joss-paper/paper.md:15]:
```suggestion
  - name: Daniel A. Holladay
  - orcid: 0000-0002-0673-9741
```

Comment (dholladay00) [reply joss-paper/paper.md:15]: @Yurlungur

File: joss-paper/paper.md [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+    affiliation: 3
+  - name: Anna Pietarila Graham
+    affiliation: 5
+  - name: Karen Tsai
+    affiliation: 3
+  - name: Brandon Barker
+    affiliation: "1, 2, 6, 7"
+  - name: Alexander Holas
+    affiliation: "2, 3, 8"
+  - name: Ann E. Wills
```
Comment (aematts) [inline joss-paper/paper.md:34]: My publishing name is Ann E. Mattsson, but you could use Ann E. Mattsson Wills if needed.

Comment (Yurlungur) [reply joss-paper/paper.md:34]: I will change to Ann E. Mattsson. Was forced to change to Ann E. Wills during RASSTI.

Comment (Yurlungur) [reply joss-paper/paper.md]: Fixed.

**Review by junghans — APPROVED** [review]
If would maybe add a sentence about the plugin framework.

**Comment by Yurlungur** [comment]
> If would maybe add a sentence about the plugin framework.

Done. Thanks for the suggestion!

**Review by junghans — APPROVED** [review]
LGTM!

**Pull request merged**